### DISCUSSION TITLE
Regarding hero.css

### DISCUSSION THREAD

**Discussion by sakshamdargan** [codersam108/Hackathon-Club-Website#5] [General]
Guys if i want to add hero.css for hero section then i would add it in components/hero only with jsx?

**Comment by MRIDUL1978** [comment]
use tailwind

**Reply by sakshamdargan** [reply]
aati nahi

**Reply by MRIDUL1978** [reply]
use gpt

**Reply by MRIDUL1978** [reply]
nahi to base bana de

**Reply by MRIDUL1978** [reply]
styling mei kr dunga ya satvik

**Comment by sanatan-dive** [comment]
> Guys if i want to add hero.css for hero section then i would add it in components/hero only with 
App.css ya index.css me css krde aur hero section ki jo jsx file hai usme upar import krdio
